StoreBroker/Helpers.ps1

```
@@ -829,7 +829,7 @@ function Resolve-UnverifiedPath
 
     $resolvedPath = Resolve-Path -Path $Path -ErrorVariable resolvePathError -ErrorAction SilentlyContinue
 
-    if ($null -eq $resolvedPath)
+    if (($null -eq $resolvedPath) -and ($resolvePathError.Count -gt 0))

```

Can you explain the scenario for this change?

---

This came about after testing with -ErrorAction Ignore.  In that case, $resolvePathError was still $null and $resolvedPath was also $null.  An exception was thrown trying to index into $resolvePathError.  After changing back to -ErrorAction SilentlyContinue, I left the additional guard in just to be safe.

I'm fine removing if you don't think we need it.

---

It's not going to help anything in this case...you'll just force it to go in to the else, and then it'll try accessing a property off of `$null` and get back an empty string, which will end up causing issues later on when the caller tries to use that "resolved" path.  Better to just let it crash here if it really doesn't have any errors that we can index into.